Cargo.toml

```diff
@@ -14,7 +14,8 @@ crate-type = ["cdylib"]
 parsepatch = "0.2"
 
 [dependencies.pyo3]
-version = "0.9"
+git = "https://github.com/Pyo3/pyo3.git"
```

Comment:
could you add a TODO comment and a link to the issue ?

Reply:
Done in c0c4d4f.